docs/changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog goes here!
 
 New features:
 
+* Add ``sphinx`` and ``sphinx_rtd_theme`` as dependencies for a new ``docs`` extra
```

Comment:
This could go under "For packagers" (if relevant) or just "Other changes" below, since it's not really a new feature for users.

Reply:
Thanks for the notice! I didn't see it, since "Other changes" is not an header :sweat_smile: